sql/core/src/main/java/org/apache/spark/sql/sources/v2/SupportsBatchWrite.java

```diff
@@ -25,14 +25,14 @@
 import org.apache.spark.sql.types.StructType;
 
 /**
- * A mix-in interface for {@link DataSourceV2}. Data sources can implement this interface to
+ * A mix-in interface for {@link Table}. Data sources can implement this interface to
  * provide data writing ability for batch processing.
  *
  * This interface is used to create {@link BatchWriteSupport} instances when end users run
```

Comment:
I don't have a better naming in mind, so I leave it as `WriteSupport` for now. Better naming is welcome to match `Scan`!

Reply:
What if we just call it BatchWrite